app/controllers/invitation_codes_controller.rb

```diff
@@ -2,7 +2,7 @@ class InvitationCodesController < ApplicationController
   before_action :ensure_valid_invite_code
 
   rescue_from ActiveRecord::RecordNotFound do
-    redirect_to root_url, :notice => "That invite code is no longer valid"
+    redirect_to root_url, :notice => I18n.t('invintation_codes.not_valid')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.